### PR TITLE
Island wings and Gare additions

### PR DESCRIPTION
**Pull request by QwindorRousseau**
Added support for the use of the Island wings in Achaea.

### CODE REVIEW

**Comment by QwindorRousseau** [comment]
Also added in support for gare which to make sure even if the player has it set to true that it wont attempt to use it if they are not dragon via gmcp.

**Comment by KaliLusternia** [comment]
Things look fairly good, though you did leave the mapping aliases turned on.
The configs were also not added to the filter list, though I'm not particularly worried about that - I plan to rewrite that once the current round of PR's goes through.

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -4817,7 +4817,7 @@ mmp.speedWalkPath = mmp.speedWalkPath or {}
 mmp.speedWalkDir = mmp.speedWalkDir or {}
 
 
-local newversion = &quot;developer&quot;
+local newversion = &quot;18.2.5&quot;
```
Comment (keneanung) [inline mudlet-mapper.xml]: Could you undo this change as well? Otherwise the release script breaks

File: mudlet-mapper.xml [hunk 2]
```diff
@@ -5662,6 +5665,18 @@ function mmp.addWingsAchaea()
...
       end
     end
+		if
+    	mmp.settings.duanatharic and
+    	gmcp.Room and
+    	not table.contains(gmcp.Room.Info.details, &quot;indoors&quot;)
+  	then
+	  	--Island wings.
+    	if mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Main&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Arcadia&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Outer&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Meropis&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Island&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;North&quot;) then
+      	if mmp.settings.duanatharic then
```
Comment (keneanung) [inline mudlet-mapper.xml]: You don't need to re-check for `mmp.settings.duanatharic` here, because it's already covered in the outer `if`.

File: mudlet-mapper.xml [hunk 3]
```diff
@@ -5712,6 +5727,23 @@ function mmp.addWingsAchaea()
       )
     end
   end
+  --gare
+  if
+    mmp.settings.gare and
+		gmcp.Room and
```
Comment (keneanung) [inline mudlet-mapper.xml]: You may want to check for `gmcp.Char` instead here.

File: mudlet-mapper.xml [hunk 4]
```diff
@@ -3443,7 +3443,7 @@ mmp.settings:setOption(matches[3], val)</script>
                 <packageName></packageName>
                 <regex>^area list$</regex>
             </Alias>
-            <AliasGroup isActive="no" isFolder="yes">
+            <AliasGroup isActive="yes" isFolder="yes">
```
Comment (keneanung) [inline mudlet-mapper.xml]: Nooooo... 😅

**Comment by QwindorRousseau** [comment]
Yea i was fixing the changes manually because doing it by script update was being a pest.
